src/Files.App/Filesystem/FilesystemOperations/FilesystemOperations.cs

```diff
@@ -111,37 +119,42 @@ public async Task<(IStorageHistory, IStorageItem)> CreateAsync(IStorageItemWithP
 						break;
 				}
 
-				errorCode?.Report(fsResult);
+                fsProgress.ProcessedItemsCount = 1;
+                fsProgress.ReportStatus(fsResult);
```

Comment:
This file could use a full reformatting by VS, I won't mark any other issue of formatting on it therefore ^^